lib/msal-common/src/auth/ScopeSet.ts

```diff
@@ -54,12 +54,12 @@ export class ScopeSet {
             if (inputScopes.length < 1) {
                 throw ClientConfigurationError.createEmptyScopesArrayError(inputScopes);
             }
-        }
 
-        // Check that clientId is passed as single scope
-        if (inputScopes && inputScopes.indexOf(this.clientId) > -1) {
-            if (inputScopes.length > 1) {
-                throw ClientConfigurationError.createClientIdSingleScopeError(inputScopes);
+            // Check that clientId is passed as single scope
+            if (inputScopes && inputScopes.indexOf(this.clientId) > -1) {
+                if (inputScopes.length > 1) {
```

Comment:
Can be one `if` statement

Reply:
I ended up removing this code in the SSO PR (not released yet), so no longer an issue.

Reply:
See PR #1211